docs/operations/azure_agents_en.md

```diff
@@ -0,0 +1,315 @@
+# Set up Jenkins agents in the Azure Cloud
```

Comment:
I guess the translated version will be made available when this document was proof-read, right?